tf-dataflow/README.md

```diff
@@ -113,7 +113,25 @@ Note that we'll be running this local experiment with just 1 app-param. Your rea
    python3 -m venv .venv
    source .venv/bin/activate
    pip install -r requirements.txt
+
+   ```
+   If you have issues setting up an environment locally, we also provide a Dockerfile that should provide a more stable environment.
+   ```bash
+   docker build . -t tfdataflow
+   # wait for container to finish building...
+
+   # Grab your credentials to use inside the docker container
+   # Example on osx:
```

Comment:
I don't know what to do with this on windows with cmd or powershell. Take a look at this guide - it might be useful in making these instructions more streamlined. http://blog.ctaggart.com/2016/11/pass-google-application-default.html

Reply:
Updated, thanks!